Orange/widgets/model/owneuralnetwork.py

```diff
@@ -135,6 +136,14 @@ def add_main_layout(self):
                 None, self, "max_iterations", 10, 10000, step=10,
                 label="Max iterations:", orientation=Qt.Horizontal,
                 alignment=Qt.AlignRight, callback=self.settings_changed))
+        
+        form.addRow(
+            "Random state:",
+            gui.spin(
+                None, self, "random_state", 0, 100000, step=20,
```

Comment:
What does `step=20` accomplish? Is the difference between 10 and 30 larger than between 10 and 11? :)

Reply:
No issue, it can be what ever value you like. ;-)
So a step size of 1 is fine, but if we drop direct entry it does not make sense anyway.